docs/howto/add_pagination_links.md

```diff
@@ -0,0 +1,46 @@
+# How to add pagination links
+
+Pagination links will be included in your response automatically as long as the resource is paginated and if you are using a ```JSON-API``` adapter. The others adapters does not have this feature.
+
```

Comment:
But you could use the PaginationLinks builder with the JSON adapter right? like if I wanted to put links in the 'meta' attribute using a subclass of the json adapter?